docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-full-du-profile/source-cr-observability.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: acm-route
+  namespace: ztp-sno-ran-du-v4-17-3
```

Comment:
Should this match the example namespace?

Reply:
Right, I missed to update this file with the example namespace. Thank you.